modules/core/shared/src/main/scala/lucuma/core/model/PosAngleConstraint.scala

```diff
@@ -20,6 +20,22 @@ sealed trait PosAngleConstraint extends Product with Serializable
 
 object PosAngleConstraint extends PosAngleConstraintOptics {
 
+  /**
+   * Specifies that the there is no constraint on the position angle.
+   * It may be set to any angle for this observation.
+   */
+  case object Unbounded extends PosAngleConstraint {
```

Comment:
We used to have in most places `Option[PosAngleConstraint]` where `None` would be used as `UNBOUNDED`
It maybe worth check on lucuma-core where this pattern has been used and fix it

Reply:
I don't know of any places in lucuma-core, but I'll look. I have another pull request coming for `lucuma-catalog`. Plus, of course, the big PR for explore.

Reply:
Right, we may encode that on catalog or explore

Reply:
The encoder is in lucuma-schemas